docs/quick-start-guide.md

```diff
@@ -62,9 +62,17 @@ import org.apache.spark.sql.CarbonSession._
 * Create a CarbonSession :
 
 ```
-val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession()
+val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession("<hdfs store path>")
 ```
 
+Or another interface as:
+
+```
+val carbon = SparkSession.builder().config(sc.getConf).getOrCreateCarbonSession("<hdfs store path>", "<hdfs
+metastore path>")
```

Comment:
@ravipesala Thanks for your comments, i have submit new pr and fix that and format code. pls help to review. thanks again.